src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -365,6 +365,10 @@ private static byte[] GetSqlServerSPN(string hostNameOrAddress, string portOrIns
             {
                 serverSpn += ":" + portOrInstanceName;
             }
+            else
+            {
+                serverSpn += ":1433";
```

Comment:
We have a default port defined in this file
private const int DefaultSqlServerPort = 1433;

Let’s use that for appending.